include/tscore/HostLookup.h

```diff
@@ -83,7 +83,7 @@ struct HostBranch {
     HOST_ARRAY,
   };
 
-  using HostTable = std::unordered_map<std::string_view, HostBranch *>;
+  using HostTable = std::unordered_map<std::string, HostBranch *>;
```

Comment:
So, it turns out that if you change this to:
```
using HostTable = std::unordered_map<std::string, HostBranch *, std::less<>>
```
then you want have to convert a string_view to a string to look it up in the table.  Since I think we need to do a host lookup for every transaction, it's worth making the change.  The definition of std::less is in the <functional> Standard Library header.

The reason is that for https://en.cppreference.com/w/cpp/container/map/find , using std::less<> as the compare parameter allows you to use overloads 3 and 4 of the find() member function.  std::string_view is valid for the K find template parameter (and the compiler can deduce it).  Seems like they should have changed std::less<> to be the default compare parameter.   I can't see how it would break existing code, but what do I know.